secure-app-model/keyvault/partnerconsent/src/main/java/com/microsoft/store/samples/partnerconsent/security/KeyVaultProvider.java

```diff
@@ -63,89 +53,24 @@ public String getSecret(String secretName)
      */
     public void setSecret(String secretName, String value)
     {
-        client.setSecret(vaultBaseUrl, secretName, value);
+        client.setSecret(secretName, value);
     }
 
-    /**
-     * Gets an access token from the authority.
-     * 
-     * @param authorization Address of the authority to issue the token.
-     * @param resource Identifier of the target resource that is the recipient of the requested token.
-     * @param clientId The identifier of the client requesting the token.
-     * @param clientSecret The secure secret of the client requesting the token.
-     * @return An instance of {@link AuthenticationResult} that contains an access token and refresh token.
-     * 
-     * @throws ExecutionException {@link ExecutionException}
-     * @throws InterruptedException {@link InterruptedException}
-     * @throws MalformedURLException {@link MalformedURLException}
-     */
-    private AuthenticationResult getAccessToken(String authorization, String resource, String clientId, String clientSecret) 
-        throws ExecutionException, InterruptedException, MalformedURLException
-    {
-        AuthenticationContext authContext; 
-        AuthenticationResult authResult;
-        ExecutorService service = null;
-        Future<AuthenticationResult> future;
-
-        try
-        {
-            service = Executors.newFixedThreadPool(1);
-            authContext = new AuthenticationContext(authorization, true, service);
-
-            future = authContext.acquireToken(
-                resource,
-                new ClientCredential(
-                    clientId, 
-                    clientSecret), 
-                null);
-            
-            authResult = future.get();
-
-            return authResult;
-        }
-        finally
-        {
-            service.shutdown();
-        }
-    }
 
     /**
      * Gets a client that is capable of interacting with the Azure Key Vault service.
-     *
-     * @param clientId The identifier of the client requesting the token.
-     * @param clientSecret The secure secret of the client requesting the token. 
      * 
      * @return A client that is capable of interacting with the Azure Key Vault service.
```

Comment:
The Secret Client, capable of managing Secrets in the Azure Key Vault by interacting with Azure Key Vault service.